admin_manual/windmill_workflows/index.rst

```diff
@@ -0,0 +1,60 @@
+==================
+Windmill Workflows
+==================
+
+Nextcloud integrates the Windmill workflow engine (https://www.windmill.dev/) to allow advanced custom workflows interacting with your Nextcloud instance.
+
+Installation
+------------
+
+ * Install Windmill
+
+   * Either as a standalone install or via the Windmill External App in Nextcloud (see :ref:`External Apps<ai-app_api>`)
+
+ * Enable the ``webhook_listeners`` app that comes with Nextcloud
+
+.. code-block:: bash
+
+   occ app:enable webhook_listeners
+
+Selecting the right Workspace
+-----------------------------
+
+When installing Windmill as an ExApp make sure to select the right workspace on the first run: Only the pre-existing "nextcloud" workspace is hooked up to nextcloud's internal event system, all other workspaces will need manual webhook setups for each workflow.
```

Comment:
```suggestion
With Windmill installed as an ExApp, the first time one visits Windmill's web interface, make sure to select the right workspace on the first run: Only the pre-existing "nextcloud" workspace is hooked up to nextcloud's internal event system, all other workspaces will need manual webhook setups for each workflow.
If the wrong workspace is selected on the first run, it is always possible to switch workspace later in the left sidebar.
```

Here is a screenshot that can demonstrate the workflow selection:
![workspace selection](https://github.com/user-attachments/assets/d0e5c335-f19e-423f-b174-4d48e20456f9)

And the selection in the sidebar:
![image](https://github.com/user-attachments/assets/a09fada8-f192-41ca-b8c6-b5c6d8d2476b)